lib/foods.js

```diff
@@ -13,34 +11,44 @@ function makeFoods(foods){
 };
 
 
-function createFoodsTable(foodsArray){
-  var foodsArray = foodsArray;
+function createFoodsTable(unsortedFoods){
+  foodsArray = sortFoodsArray(unsortedFoods);
   foodsArray.forEach(function(food){
-    $(".food-table").append("<tr class='tbl-row'><td class='cell-one food-cell' contenteditable='true'>"
-    + "<span class='replaceme-name' contenteditable='true'>"
-    + food.name
-    + "</span>"
-    + "</td><td class='cell-two cal-cell'>"
-    + "<span class='replaceme-cal' contenteditable='true'>"
-    + food.calories
-    + "</span>"
-    + "</td><td class='delete-cell'><button class='delete-food-btn'><img class='delete-food-img' src='lib/images/delete_red.png'/></button></td><td class='food-id' style='visibility:hidden;'>"
-    + food.id
-    + "</td></tr>");
+    $(".food-table").append(foodString(food))
   });
```

Comment:
Pulled out the string append into its own method

Reply:
Thank goodness 💯 that thing was a monster